### PR TITLE
Add e2e test from eventing-kafka

### PR DESCRIPTION
**Pull request by creydr**
This PR adds the e2e tests which ran in eventing-kafka but not in eventing-kafka-broker

### CODE REVIEW

File: test/e2e_source/kafka_binding_test.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import (
+	"testing"
+
+	"knative.dev/eventing-kafka-broker/test/e2e_source/helpers"
+)
+
+func TestKafkaBinding(t *testing.T) {
+	helpers.TestKafkaBinding(t)
+}
```
Comment (pierDipi) [inline test/e2e_source/kafka_binding_test.go:30]: We don't support KafkaBinding in this repository

File: test/e2e_source/kafka_source_claims_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import (
+	"testing"
+
+	"knative.dev/eventing-kafka-broker/test/e2e_source/helpers"
+)
+
+func TestKafkaSourceClaims(t *testing.T) {
+	helpers.TestKafkaSourceClaims(t)
+}
```
Comment (pierDipi) [inline test/e2e_source/kafka_source_claims_test.go:30]: I think Claims are not supported for this new Source implementation

File: test/e2e_source/kafka_binding_test.go (hunk 1 above)
Comment (creydr) [reply test/e2e_source/kafka_binding_test.go]: Ah ok :+1:

**Comment by creydr** [comment]
Closing this, as:

1. Source - KafkaBinding Test: KafkaBindings are not supported in this repo anymore
2. Source Claims Test: Claims are not supported by the new Source implementation
3. Source Reconciler Tests: Requires the IMC installed and targets the old implementation and status conditions (https://knative.slack.com/archives/C02C56QF282/p1674208618123049) -> this could be worth a follow up to change the current implementation to set a better condition reason
4. Broker Channel Test: Requires the MTChannelBasedBroker installed

**Pull request closed without merging**